source/get-started/virtual-machine-install/virt-manager.rst

```diff
@@ -60,13 +60,11 @@ This example uses the live-server-installer ISO to create a new installation.
    file hash and signatures. Refer to :ref:`validate-signatures` for detailed
    steps.
 
-Launch and setup virt-manager
-*****************************
-
-Virt-manager is a GUI-based virtual machine manager that runs in your desktop
-environment. I'm using the Gnome desktop and will launch the Virtual Machine
-manager from the applications window. You will see the following application
-window:
+Next, we'll set up the virt-manager in preparation to load the ISO from the
```

Comment:
Minor point. I liked this header you had previously. I meant to add a transitional sentence for context, but not to remove the header. Do you mind adding it back in?

Reply:
Did you want the transitional sentence as the final sentence of the previous section and then have the header?  I saw the - in front of the header so thought you wanted that removed.

Reply:
Hopefully this is what you meant...